features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/blocks/api-category/api-category-list/ajax/api-category-list.jag

```diff
@@ -0,0 +1,36 @@
+<%
+include("/jagg/jagg.jag");
+
+(function () {
+    response.contentType = "application/json; charset=UTF-8";
+    var mod, result, obj,
+    action = request.getParameter("action"),
+    site = require("/site/conf/site.json"),
+    msg = require("/site/conf/ui-messages.jag");
+    if(jagg.getUser() == null){
```

Comment:
Formatting issue

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/8077/commits/c6c0ee00e2fc4d641567d19b696b59a2a92f077b